source/assets/stylesheets/sections/_railsgirls.scss

```diff
@@ -0,0 +1,18 @@
+.rectangle {
+  width: 250px;
+  height: 60px;
+  background: $brand-primary;
+  margin: 0 auto;
+  line-height: 60px;
+}
+
+.social-links {
+  width: 200px;
+  margin: 0 auto;
+}
+
+.imgsize2 {
+  resize:both;
+  max-width: 500px;
+  height: 400px;
```

Comment:
This is a code smell. you usually dont need to set the height of a repsonsive image since it will adapt to the width